LayoutTests/TestExpectations

```diff
@@ -1891,6 +1891,13 @@ imported/w3c/web-platform-tests/mathml/presentation-markup/operators/mo-form-dyn
 # This MathML test should be rewritten.
 webkit.org/b/201356 mathml/presentation/stretchy-depth-height-symmetric.html [ Skip ]
 
+# These internal MathML tests are outdated and do not match MathML Core.
+mathml/mathml-unknown.xhtml [ Failure ]
+mathml/presentation/foreign-element-in-annotation-xml.html [ Failure ]
+mathml/presentation/semantics-2.html [ ImageOnlyFailure ]
+mathml/presentation/semantics-3.html [ ImageOnlyFailure ]
+mathml/presentation/semantics-4.html [ ImageOnlyFailure ]
+
```

Comment:
I think we will eventually remove these tests when we align semantics/maction with MathML Core, but I don't understand why they are failing with your change. Can you explain why? Maybe there is a way to tweak them so they still pass for now.